### PR TITLE
add actions and issue templates

### PR DESCRIPTION
**Pull request by Poorunga**
1. add actions
2. add issue templates

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/question.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+name: Question
+about: Question relating to EdgeMesh.
+labels: kind/question
```
Comment (kevin-wangzefeng) [inline .github/ISSUE_TEMPLATE/question.md:4]: For questions, now we can use https://github.com/kubeedge/kubeedge/discussions instead.

Comment (Poorunga) [reply .github/ISSUE_TEMPLATE/question.md]: fix

File: .github/ISSUE_TEMPLATE/question.md [hunk 2]
```diff
@@ -0,0 +1 @@
+For questions, now we can use https://github.com/kubeedge/kubeedge/discussions
```
Comment (kevin-wangzefeng) [inline .github/ISSUE_TEMPLATE/question.md:1]: This doesn't block people from creating questions as issues.
You may refer to: https://github.com/kubernetes/kubernetes/blob/master/.github/ISSUE_TEMPLATE/config.yml

Comment (Poorunga) [reply .github/ISSUE_TEMPLATE/question.md]: Good suggestion.

Comment (Poorunga) [reply .github/ISSUE_TEMPLATE/question.md]: fix

**Pull request merged**